redash/query_runner/cass.py

```diff
@@ -54,22 +54,36 @@ def type(cls):
         return "Cassandra"
 
     def _get_tables(self, schema):
-        query = """
+        CF_query = """
         select columnfamily_name from system.schema_columnfamilies where keyspace_name = '{}';
         """.format(self.configuration['keyspace'])
 
-        results, error = self.run_query(query, None)
-        return results, error
+        results, error = self.run_query(CF_query, None)
+        results = json.loads(results)
+        for row in results['rows']:
+            table_name = row['columnfamily_name']
+            if table_name not in schema:
+                schema[table_name] = {'name': table_name, 'columns': []}
+            CN_query = """
+            SELECT column_name FROM system.schema_columns where keyspace_name ='{}' and columnfamily_name ='{}';
+            """.format(self.configuration['keyspace'], table_name)
```

Comment:
Why not run: `SELECT columnfamily_name, column_name FROM system.schema_columns where keyspace_name ='{}'` and split them to column families in the code? This way you need to run only a single query instead of N queries.